src/commands/CommandPage.vala

```diff
@@ -219,9 +224,14 @@ namespace Ilia {
             // TODO ~ Enable two launch modes with some modifier.  By default terminal exits when program exits.
             // todo is to add another mode in which the terminal does not exit after program completes.
 
-            // string commandline = "/usr/bin/x-terminal-emulator -e \"bash -c '" + cmd_path + "; exec bash'\"";
-            // string commandline = "x-terminal-emulator -e \"bash -c '" + cmd_path + "'\"";
-            string commandline = "x-terminal-emulator -e '" + cmd_path + "'";
+            string commandline;
+            if (this.quiet)
+                // if -q is given, call cmd_path through bash and ignore output
+                commandline = "bash -c '" + cmd_path + " > /dev/null 2>&1'";
```

Comment:
concern: this adds an implicit dependency on `bash` and may not be the termail preference of the user.  We could add a dependency on bash [here](https://github.com/regolith-linux/ilia/blob/main/debian/control#L28), but I wonder if it's possible to achieve the goal without requiring bash?  Can we not use `x-terminal-emulator` or some similar abstraction over the user's terminal preference?

Reply:
I came up with two approaches:

1. **[`ceb3dbe`](https://github.com/regolith-linux/ilia/pull/96/commits/ceb3dbee729d51e2126f011cef20bdb2b6b8bfc8)**:  
   This solution uses `nohup`, which aligns with the default behavior of i3 and is part of `coreutils`. This approach is robust and avoids output in the invoking console.

2. **[`b49f0f8`](https://github.com/regolith-linux/ilia/pull/96/commits/b49f0f8f643e457be224265268e07baca56537f7)**:  
   This alternative directly executes the program. While it avoids adding a dependency on `coreutils`, it does result in output being shown in the console invoking `ilia`. This should be acceptable since `ilia` is typically invoked from a command line.

Do you think one of those is better?

Reply:
Sorry for the delay @PeterNaggschga .  I like your pick.